Src/PCompiler/CompilerCore/Backend/CSharp/CSharpCodeGenerator.cs

```diff
@@ -263,7 +263,7 @@ private void WriteMonitor(CompilationContext context, StringWriter output, Machi
             WriteNameSpacePrologue(context, output);
 
             var declName = context.Names.GetNameForDecl(machine);
-            context.WriteLine(output, $"internal partial class {declName} : PMonitor");
+            context.WriteLine(output, $"internal partial class {declName} : PChecker.Specifications.Monitors.Monitor");
```

Comment:
Instead of this; add `using PChecker.Specifications.Monitors;` and then just have the `Monitor`